sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIAuthenticationAudience.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+using static System.Net.WebRequestMethods;
+
+namespace Azure.AI.OpenAI;
+
+/// <summary>
+/// Represents cloud authentication audiences available for Azure OpenAI.
+/// These audiences correspond to authorization token authentication scopes.
+/// </summary>
+public readonly partial struct AzureOpenAIAuthorizationAudience : IEquatable<AzureOpenAIAuthorizationAudience>
```

Comment:
This should be called "AzureOpenAIAudience" to align with the pattern used in the other Azure libraries such as Text Analytics, Search, Monitor, Storage, etc.